### ISSUE TITLE
error

### ISSUE THREAD

**Issue by eyzed367** [Technowlogy-Pushpender/crypter#5]
i have a problem 

i crypting exe files with this crypter and run it and give me error

error is :

this app can't run on your pc

why?

**Comment by Technowlogy-Pushpender** [comment]
This is not an exe crypter, it can crypt python source codes, if you open
your final exe you will notice that it contains python obfuscated source
codes with encrypted hex codes.

Please read the documentation before starting a new issue
